src/io/flutter/run/bazelTest/BazelTestLaunchState.java

```diff
@@ -106,8 +106,8 @@ protected ConsoleView createConsole(@NotNull Executor executor) throws Execution
     final ConsoleProps props = ConsoleProps.forBazel(config, executor, resolver);
     final BaseTestsOutputConsoleView console = SMTestRunnerConnectionUtil.createConsole(ConsoleProps.bazelFrameworkName, props);
 
-    final String baseDir = workspace.getRoot().getPath();
-    console.addMessageFilter(new DartRelativePathsConsoleFilter(project, baseDir));
```

Comment:
DartRelativePathsConsoleFilter is intended to be used in test environs. Are you sure deleting it is the right thing to do?